src/screens/Inventory/PurchaseRequest.tsx

```diff
@@ -78,7 +81,7 @@ function PurchaseRequest(props: PurchaseRequestsProps) {
   };
 
   return (
-    <BaseScreen title="Inventory Receipt" leftIcon="backNav">
+    <BaseScreen title={intl(words.inventory_purchase)} leftIcon="backNav">
```

Comment:
inventory receipt not inventory purchase, though I guess we don't have a translation for this?

Reply:
Just added one. NPO will need to translate it though.